spinnaker/spinnaker_system/google_http_upsert_test.py

```diff
@@ -0,0 +1,891 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Could you rename this module so it more clearly relates to load balancing (even lb)?